record-editor/src/app/record-editor/record-toolbar/record-toolbar.component.ts

```diff
@@ -133,19 +134,22 @@ export class RecordToolbarComponent extends SubscriberComponent
 
   private cleanupAndSaveRecord(record) {
     this.recordCleanupService.cleanup(record);
-    this.apiService
-      .saveRecord(record)
-      .subscribe(() => this.onSaveSuccess(), error => this.onSaveError(error));
+    this.apiService.saveRecord(record).subscribe(
+      () => this.onSaveSuccess(),
+      (error) => this.onSaveError(error)
+    );
   }
 
   private onSaveSuccess() {
     this.domUtilsService.unregisterBeforeUnloadPrompt();
-
-    if (this.pidType === 'conferences') {
+    this.toastrService.success('Changes are saved', 'Success');
+    if (this.pidType === 'conferences' || this.pidType === 'literature') {
```

Comment:
what about authors?

Reply:
yeah, we want this behavior for any pid type.

Reply:
Updating 👍